mongo/mongo_test.go

```diff
@@ -116,7 +116,7 @@ func TestMongoHelpers(t *testing.T) {
 			got, id, err := transformAndEnsureID(bson.DefaultRegistry, doc)
 			assert.Nil(t, err, "transformAndEnsureID error: %v", err)
 			_, ok := id.(string)
-			assert.True(t, ok, "expected returned id type %T, got %T", string(0), id)
+			assert.True(t, ok, "expected returned id type string, got %T", id)
```

Comment:
What is this change from?

Reply:
this change was from either the 1.15 linter or compiler getting mad. the 1.4 branch has the new phrasing on it